src/testing/utils.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+/**
+ * Assert that some JSX from Enzyme is equal to some provided JSX.
+ * @param {Object} actual - Some JSX from Enzyme.
+ * @param {Object} expected - Some JSX provided in the test.
+ */
+export const compareJSX = (actual, expected) => {
```

Comment:
Were these files just copied over? I am not sure why we'd use this vs snapshots?

Reply:
17:30:37 <barrymcgee> Hatch yes, those tests were just copied over verbatim.. I spent a while trying to cherry pick only the bits I needed then figured if we’re gonna hopefully abstract to a lib later down the line, I shouldn’t diverge without good reason